src/airflow/run-airflow-only.sh

```diff
@@ -1,4 +1,8 @@
 #!/bin/bash
 
 export DOCKER_BUILDKIT=1
+docker-compose -f docker-compose.yaml up --build db db-setup webserver scheduler
+
+#WINDOW
+set DOCKER_BUILDKIT=1
```

Comment:
Hi @wabinyai, thanks for this. 
One change though, this is just a bash script  & not the microservice's  README, so what you'll do is indicate all these lines  as  comments i.e . 
```
#For windows users
#steps to follow
```